custom_components/frigate/__init__.py

```diff
@@ -454,8 +449,8 @@ async def async_will_remove_from_hass(self) -> None:
         self._sub_state = None
 
     @callback  # type: ignore[misc]
-    def _state_message_received(self, msg: ReceiveMessage) -> None:
-        """State message received."""
+    def _update_message_received(self, msg: ReceiveMessage) -> None:
```

Comment:
Since there's no even pretense of commonality now, and since it's conceivable the topic_map will contain messages that don't require a HA state, I recommend removing this method entirely and just call `self.async_write_ha_state()` whenever you need to (i.e. wherever `_update_message_received` is called today).